Applications/Inc/fifo.h

```diff
@@ -0,0 +1,223 @@
+// Copyright (c) 2021 Longhorn Racing Solar
```

Comment:
Although I didn't end up using it, I think this file should still be kept just in case we use it later on for another purpose.

Reply:
Agreed. Does no harm to have this around.